CONTRIBUTING.md

```diff
@@ -156,7 +156,12 @@ def test_add_one_and_one_equals_two():
 
 #### Integration tests
 
-Set environment variables C42_USERNAME and C42_PW with CCA credentials. To execute integration tests:
+If not using the mock server, set the environment variables `C42_USER` and `C42_PW`
+with CCA credentials. Otherwise, the integration tests default to using
+`127.0.0.1:4200`, which is the same address that the mock server is set to run on.
```

Comment:
Should we make host be an env variable too? Or at least call out where you'd change the host from the default here.

Reply:
Done